File: app/src/main/java/com/example/exitpro/Model/LateStudent.java
package com.example.exitpro.Model;

public class LateStudent {
    private String name;
    private int rollNumber;
    private String destination;
    private String phoneNumber;
    private int year;
    private int month;
    private int day;
    private String hour;
    private String minute;
    private String second;

    // Getters and setters for each attribute

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getRollNumber() {
        return rollNumber;
    }

    public void setRollNumber(int rollNumber) {
        this.rollNumber = rollNumber;
    }

    public String getDestination() {
        return destination;
    }

    public void setDestination(String destination) {
        this.destination = destination;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public int getYear() {
        return year;
    }

    public void setYear(int year) {
        this.year = year;
    }

    public int getMonth() {
        return month;
    }

    public void setMonth(int month) {
        this.month = month;
    }

    public int getDay() {
        return day;
    }

    public void setDay(int day) {
        this.day = day;
    }

    public String getHour() {
        return hour;
    }

    public void setHour(String hour) {
        this.hour = hour;
    }

    public String getMinute() {
        return minute;
    }

    public void setMinute(String minute) {
        this.minute = minute;
    }

    public String getSecond() {
        return second;
    }

    public void setSecond(String second) {
        this.second = second;
    }

    @Override
    public String toString() {
        return "LateStudent{" +
                "name='" + name + '\'' +
                ", rollNumber=" + rollNumber +
                ", destination='" + destination + '\'' +
                ", phoneNumber='" + phoneNumber + '\'' +
                ", year=" + year +
                ", month=" + month +
                ", day=" + day +
                ", hour='" + hour + '\'' +
                ", minute='" + minute + '\'' +
                ", second='" + second + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/exitpro/Model/LateStudent.kt
package com.example.exitpro.Model

class LateStudent {
    // Getters and setters for each attribute
    var name: String? = null
    var rollNumber: Int = 0
    var destination: String? = null
    var phoneNumber: String? = null
    var year: Int = 0
    var month: Int = 0
    var day: Int = 0
    var hour: String? = null
    var minute: String? = null
    var second: String? = null

    override fun toString(): String {
        return "LateStudent{" +
                "name='" + name + '\'' +
                ", rollNumber=" + rollNumber +
                ", destination='" + destination + '\'' +
                ", phoneNumber='" + phoneNumber + '\'' +
                ", year=" + year +
                ", month=" + month +
                ", day=" + day +
                ", hour='" + hour + '\'' +
                ", minute='" + minute + '\'' +
                ", second='" + second + '\'' +
                '}'
    }
}
